Homework-8/test-1.py

```diff
@@ -0,0 +1,18 @@
+text_input = input("Enter string: ")
+
+# with for loop
+
+# for char in range(len(text_input)):
+#     if char % 2 == 0:
+#         if text_input[char] != "e":
+#             print(text_input[char], end="")
+
+# with while loop
+
+index = 0
+while index < len(text_input):
```

Comment:
პირდაპირ მხოლოდ ლუწ ინდექსებს გადაყოლოდი შეგეძლო for i in range(0, len(text_input), 2)-ით